python/mxnet/io/io.py

```diff
@@ -966,6 +988,164 @@ def creator(*args, **kwargs):
     creator.__doc__ = doc_str
     return creator
 
+
+def _read_list(list_file, batch_size):
+    """
+    Helper function that reads the .lst file, binds it in
+    a generator and returns a batched version of the generator.
+    Parameters
+    ----------
+    list_file: input list file.
```

Comment:
convention for mxnet docs:

Parameters
-----------
arg_name : type
    Explanations
arg_name1: type1
    Explanations1

Reply:
i will update the doc strings to render properly.